packages/server/src/fhir/bulkdata.ts

```diff
@@ -46,7 +53,16 @@ bulkDataRouter.get(
   })
 );
 
-bulkDataRouter.delete('/export/:id', (req: Request, res: Response) => {
+bulkDataRouter.delete('/export/:id', async (req: Request, res: Response) => {
+  const { id } = req.params;
+  const bulkDataExport = await getExportResource(id);
```

Comment:
At some point in the near future, this should probably call the `AsyncJob/$cancel` handler like the analogous DELETE route for the `AsyncJob` route